### PR TITLE
Exclude banned dependency commons-logging in the BOM 

### PR DESCRIPTION
**Pull request by ppalaga**
#30813 is related but this does not fix it because it does not add any automatic solution to add exclusion for all (or selected) banned dependencies.

### CODE REVIEW

**Comment by ppalaga** [comment]
Do I need to go through the test failures?

**Comment by Sanne** [comment]
Looks useful, not sure why nobody noticed. Would you mind resolving the conflicts and re-push?

File: bom/application/pom.xml [hunk 1]
```diff
@@ -4168,6 +4168,12 @@
                 <groupId>org.apache.httpcomponents</groupId>
                 <artifactId>httpclient-cache</artifactId>
                 <version>${httpclient.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>commons-logging</groupId>
+                        <artifactId>*</artifactId>
```
Comment (gsmet) [inline bom/application/pom.xml:4174]: So this change was apparently already partially applied (see the artifact just above) but with a wildcard so I kept the wildcard for this one too.

**Review by gsmet — APPROVED** [review]
Let's see what CI has to say.

**Comment by ppalaga** [comment]
Thanks for taking care, @gsmet!

**Comment by ppalaga** [comment]
It's never too late, thanks @gastaldi for taking this over!

**Pull request merged**